packages/components/src/tabs/src/TabLine.tsx

```diff
@@ -19,34 +21,41 @@ export function TabLine(props: TabLineProps) {
     const { selectedTab, disabledKeys, isDisabled: isTabsDisabled } = props;
     const state = useContext(TabListStateContext);
     const [isDisabled, setIsDisabled] = useState<boolean>(false);
+    const { size } = useContext(InternalTabsContext) ?? {};
 
     useEffect(() => {
         const disabled = isTabsDisabled || isAllTabsDisabled(state?.collection, disabledKeys ? new Set(disabledKeys) : new Set(null));
 
         setIsDisabled(disabled);
     }, [state?.collection, disabledKeys, isTabsDisabled, setIsDisabled]);
 
-    const [style, setStyle] = useState<{ transform: string | undefined; width: string | undefined; height: string | undefined }>({
+    const [style, setStyle] = useState<{ transform?: string ; width?: string }>({
         transform: undefined,
-        width: undefined,
-        height: undefined
+        width: undefined
     });
 
     const onResize = useCallback(() => {
         if (selectedTab) {
-            const styleObj: { transform: string | undefined; width: string | undefined; height: string | undefined } = {
+            const styleObj: { transform?: string; width?: string } = {
                 transform: undefined,
-                width: undefined,
-                height: undefined
+                width: undefined
             };
 
+            let tablistPadding = "var(--hop-space-inset-xs)";
```

Comment:
could we get those value from getComputedValue on the tablist and tab object? this way if we change the CSS it is reflected here?